bin/steps/python

```diff
@@ -4,6 +4,21 @@ set +e
 runtime-fixer runtime.txt
 PYTHON_VERSION=$(cat runtime.txt)
 
+if [[ $PYTHON_VERSION == *"*"* ]]; then
+  if [[ $PYTHON_VERSION == *"3.8"* ]]; then
```

Comment:
I think this would cause false positives - `$PYTHON_VERSION == *"3.8"*` will match against versions like `*.3.8` or `2.3.8*`